client/src/modules/user/UserProfile/EmailReports.jsx

```diff
@@ -79,6 +79,12 @@ class EmailReports extends PureComponent {
     this.setState({ emailReportsFrequency: value }, this.updateReportSettings);
   };
 
+  handleReports = () => {
+    const { prepareReport } = this.props;
+
+    prepareReport();
```

Comment:
I think this should be `sendReport`, because at the end it sends a report to the user. Preparing items for the report is not main goal of this action